sdk/identity/identity-spa/README.md

```diff
@@ -0,0 +1,228 @@
+## Azure Identity Plugin for Single Page Applications
+
+This package provides two new credentials for the Azure Identity library for JavaScript ([`@azure/identity`](https://npmjs.com/package/@azure/identity)) called `RedirectCredential` and `PopupCredential`. Both credentials simplify the browser development experience of applications working with Azure services.
+
+Both the `RedirectCredential` and the `PopupCredential` use the [Authorization Code Flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow) to authenticate server-side applications with browser user interaction.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity-spa) | [Samples](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity-spa/samples-dev)
+
+## Getting started
+
+An example using the `RedirectCredential`:
+
+```javascript
+const { RedirectCredential } = require("@azure/identity-spa");
+const { KeyClient } = require("@azure/keyvault-keys");
+
```

Comment:
TODO: Mention here how the onPageLoad function plays a role before the credential is used.